moveit_planners/chomp/chomp_motion_planner/src/chomp_cost.cpp

```diff
@@ -95,6 +95,9 @@ double ChompCost::getMaxQuadCostInvValue() const
 
 void ChompCost::scale(double scale)
 {
+  if (scale == 0.0)
```

Comment:
When scale is almost 0, is this processing OK?

Reply:
The idea here is that `max_cost_scale` enters scale without being updated, so we will deal with the above `if` statement. If `max_cost_scale` is updated, the design value (value range) is unknown, so no action will be taken.

Reply:
I understand your policy.